docs/content/latest/drivers-orms/csharp/postgres-npgsql.md

```diff
@@ -0,0 +1,136 @@
+
+## Step 3: Query the YugabyteDB Cluster from Your Application
+
+Next, copy the sample code below in the Program.cs file in order to setup a YugbyteDB Tables and query the Table contents from the java client. Ensure you replace the connection string `yburl` with credentials of your cluster and SSL certs if required.
+
+```csharp
+using System;
+using Npgsql;
+
+namespace Yugabyte_CSharp_Demo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            NpgsqlConnection conn = new NpgsqlConnection("host=localhost;port=5433;database=yb_demo;user id=yugabyte;password=");
+
+            try
+            {
+                conn.Open();
+
+                NpgsqlCommand empCreateCmd = new NpgsqlCommand("CREATE TABLE employee (id int PRIMARY KEY, name varchar, age int, language varchar);", conn);
+                empCreateCmd.ExecuteNonQuery();
+                Console.WriteLine("Created table Employee");
+
+                NpgsqlCommand empInsertCmd = new NpgsqlCommand("INSERT INTO employee (id, name, age, language) VALUES (1, 'John', 35, 'CSharp');", conn);
+                int numRows = empInsertCmd.ExecuteNonQuery();
+                Console.WriteLine("Inserted data (1, 'John', 35, 'CSharp')");
+
+                NpgsqlCommand empPrepCmd = new NpgsqlCommand("SELECT name, age, language FROM employee WHERE id = @EmployeeId", conn);
+                empPrepCmd.Parameters.Add("@EmployeeId", NpgsqlTypes.NpgsqlDbType.Integer);
+
+                empPrepCmd.Parameters["@EmployeeId"].Value = 1;
+                NpgsqlDataReader reader = empPrepCmd.ExecuteReader();
+
+                Console.WriteLine("Query returned:\nName\tAge\tLanguage"); 
+                while (reader.Read())
+                {
+                    Console.WriteLine("{0}\t{1}\t{2}", reader.GetString(0), reader.GetInt32(1), reader.GetString(2));
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Failure: " + ex.Message);
+            }
+            finally
+            {
+                if (conn.State != System.Data.ConnectionState.Closed)
+                {
+                    conn.Close();
+                }
+            }
+        }
+    }
+}
+```
+
+When you run the Project, it should output something like below:
+
+```output
+Created table Employee
+Inserted data (1, 'John', 35, 'CSharp')
+Query returned:
+Name  Age  Language
+John  35   CSharp
+```
+
+if you receive no output or error, check whether you included the proper connection string in your Program.cs file with the right credentials.
+
+After completing this steps, you should have a working C# app that uses YugabyteDB JDBC driver for connecting to your cluster, setup tables, run query and print out results.
```

Comment:
This summary would be more useful at the start/top of this document - all the examples should have a short paragraph describing what the user will be building